modules/programs/sqlite.nix

```
@@ -0,0 +1,133 @@
+{
+  lib,
+  config,
+  pkgs,
+  ...
+}:
+
+let
+  cfg = config.programs.sqlite;
+  defaultColSeparator = "|";
+  defaultPrompt = "sqlite> ";
+in
+{
+  meta.maintainers = [ lib.hm.maintainers.fk29g ];
+
+  options.programs.sqlite = {
+    enable = lib.mkEnableOption "sqlite";
+
+    package = lib.mkPackageOption pkgs "sqlite" { nullable = true; };
```

Assuming this is intended for interactive use?

```suggestion
    package = lib.mkPackageOption pkgs "sqlite-interactive" {
      nullable = true;
      example = [ "sqlite" ];
    };
```